src/main/java/dan200/computercraft/client/render/TileEntityTurtleRenderer.java

```diff
@@ -51,24 +50,7 @@
     private static ModelResourceLocation ADVANCED_TURTLE_MODEL = new ModelResourceLocation( "computercraft:turtle_advanced", "inventory" );
     private static ModelResourceLocation COLOUR_TURTLE_MODEL = new ModelResourceLocation( "computercraft:turtle_white", "inventory" );
     private static ModelResourceLocation BEGINNER_TURTLE_MODEL = new ModelResourceLocation( "computercraftedu:CC-TurtleJunior", "inventory" );
-    private static ModelResourceLocation[] BEGINNER_TURTLE_COLOUR_MODELS = new ModelResourceLocation[] {
-        new ModelResourceLocation( "computercraftedu:turtleJunior_black", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_red", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_green", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_brown", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_blue", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_purple", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_cyan", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_lightGrey", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_grey", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_pink", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_lime", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_yellow", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_lightBlue", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_magenta", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_orange", "inventory" ),
-        new ModelResourceLocation( "computercraftedu:turtleJunior_white", "inventory" ),
-    };
+    private static ModelResourceLocation BEGINNER_TURTLE_COLOUR_MODEL = new ModelResourceLocation( "computercraftedu:turtleJunior_white", "inventory" );
```

Comment:
Ideally the API could be powerful enough that there didn't have to be any special support. I'm not sure how feasible that is though.